src/CheckoutSdk/Reports/IReportsClient.cs

```diff
@@ -8,5 +9,8 @@ public interface IReportsClient
         Task<ReportsResponse> GetAllReports(ReportsQuery query, CancellationToken cancellationToken = default);
 
         Task<ReportDetailsResponse> GetReportDetails(string reportId, CancellationToken cancellationToken = default);
+
+        Task<ContentsResponse> GetReportFile(string reportId, string fileId,
+            CancellationToken cancellationToken = default);

```

Comment:
NIT: this could fit on previous line

Reply:
![image](https://user-images.githubusercontent.com/127134616/227900154-e562cbc3-10be-4d54-821a-385956123e1e.png)